bot/exts/moderation/verification.py

```diff
@@ -55,7 +55,7 @@
 """
 
 ALTERNATE_VERIFIED_MESSAGE = f"""
-Thanks for accepting our rules!
+You're now verified!
```

Comment:
It may just be me, but I feel this would be "easier" to read for more users by saying 
```suggestion
You are now verified!
```

Reply:
I made that change under the pretense that we'd still have bot verification. Does anyone actually get the alternate verified message anymore?